### DISCUSSION TITLE
Any way to set up a "rewind on pause"?

### DISCUSSION THREAD

**Discussion by MayNoii** [MusicPlayerDaemon/MPD#2054] [Q&A]
So I'm experiencing #1136, and it is pretty much the only thing preventing me from adopting MPD as my go-to offline music player. While I'm not well versed in the technical difficulties in fixing that (which I assume there are plenty, considering how long it's been standing unfixed), I was wondering if there was any way to configure MPD to rewind some time every time the player is paused (or resumed from a pause)? That way I could rewind, say, 0.2s every time I pause a song, effectively nullifying the annoying skip.

This is very easy to do with shell scripting and `mpc` (with the caveat of `mpc seek` not allowing absolute intervals smaller than one second, but this rewind is still less annoying than the skip), but of course that doesn't apply universally - pausing inside a player or using MPRIS controls won't work with that strategy, for example.

**Comment by naglis** [comment]
Wish I had a better answer to this. I ended up doing this client-side, i.e. when unpausing the client would first seek backwards.

Another very hacky option could be to have a dedicated client that would `idle` for `player` events and do the seek backwards immediately once it detects that MPD is paused so that other clients (players) would not need to care about it.
